jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+`myorb/foo: { dostuff: false }`.
+-->
+
+These conditions are checked before a workflow is actually run. That
+means, for example, that a user can't use a condition to check an environment
+variable.
+
+Conditional steps can be located anywhere a regular step could. 
+
+<!---
+For example, an
+orb author could define conditional steps in the `steps` key of a Job or a
+Command.
+-->
+
+A conditional step consists of a step with the key `when` or `unless`. Under this conditional key are the subkeys `steps` and `condition`. If `condition` is met (using when/unless logic), the subkey `steps` are run. 
+
+A `condition` is a single value that evaluates to `true` or `false` at the time the config is processed, so you cannot use environment variables as conditions, as those are not injected until your steps are running in the shell of your execution environment. You may use parameters as your conditions. The empty string will resolve as falsey in `when` conditions.
+
+### Example
+{:.no_toc}
+
+```yaml
+# Contents of the orb `myorb` in namespace `mynamespace`
+jobs:
+  myjob:
+    parameters:
+      preinstall-foo:
+        type: boolean
+        default: false
+    machine: true
+    steps:
+      - run: echo "preinstall is << parameters.preinstall-foo >>"
+      - when:
+          condition: << parameters.preinstall-foo >>
+          steps:
+            - run: echo "preinstall"
+      - unless:
+          condition: << parameters.preinstall-foo >>
+          steps:
+            - run: echo "don't preinstall"
+```
+
+```yaml
+# inside config.yml
+version: 2.1
+
+orbs:
+  myorb: mynamespace/myorb@1.0.1
+
+workflows:
+  workflow:
+    jobs:
+      - myorb/myjob:
+          preinstall-foo: false
+      - myorb/myjob:
+          preinstall-foo: true
+```
+
+
+
+## Authoring and Using Executors 
+**Note:** Reusable `executor` declarations are available in configuration version 2.1 and later.
+
+Executors define the environment in which the steps of a job will be run. When declaring a `job` in CircleCI configuration, you define the type of execution environment (`docker`, `machine`, `macos`. etc.) to run in, as well as any other parameters of that environment including: environment variables to populate, which shell to use, what size `resource_class` to use, etc. 
+
+Executor declarations in config outside of `jobs` can be used by all jobs in the scope of that declaration, allowing you to reuse a single executor definition across multiple jobs.
+
+An executor definition includes the subset of the children keys of a `job` declaration related to setting the environment for a job to execute. This means it does **not** include `steps`. That subset is one or more of the following keys:
+
+- `docker` or `machine` or `macos` 
+- `environment`
+- `working_directory`
+- `shell`
+- `resource_class`
+
+A simple example of using an executor:
+
+```yaml
+version: 2.1
+executors:
+  my-executor:
+    docker:
+      - image: circleci/ruby:volatile
```

Comment:
change "volatile" to "2.5.1-node-browsers" here -- volatile is our convention in orbs but not in our docker images

Reply:
thanks, fixed